controllers/userController.js

```diff
@@ -1,19 +1,42 @@
-const userModel = require('../data/model')
```

Comment:
I see that you changed the name of the model file to service. I also think it is advisable to change the name of the variable to userService